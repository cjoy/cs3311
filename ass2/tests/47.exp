Timothy Cappie-Wood (3363921)
Suggestions for study in 11s2

Course   Title                 UOC Requirement
-------- -----                ---- -----------
COMP3711 Software Project Man    6 Level 3 core
COMP3421 Computer Graphics       6 Comp Game core
COMP3151 Foundations of Concu    6 Comp Sci Electives
COMP3161 Concepts of Programm    6 Comp Sci Electives
COMP3171 Object-Oriented Prog    6 Comp Sci Electives
COMP3222 Digital Circuits and    6 Comp Sci Electives
COMP3331 Computer Networks&Ap    6 Comp Sci Electives
COMP3431 Robotic Software Arc    6 Comp Sci Electives
COMP3441 Security Engineering    6 Comp Sci Electives
COMP3511 Human Computer Inter    6 Comp Sci Electives
COMP3601 Design Project A        6 Comp Sci Electives
COMP3901 Special Project A       6 Comp Sci Electives
COMP3902 Special Project B      12 Comp Sci Electives
COMP4001 Object-Oriented Soft    6 Comp Sci Electives
COMP4904 Industrial Training    18 Comp Sci Electives
COMP4930 Thesis Part A           6 Comp Sci Electives
COMP9000 Special Program         6 Comp Sci Electives
COMP9311 Database Systems        6 Comp Sci Electives
COMP9321 Web Applications Eng    6 Comp Sci Electives
COMP9323 e-Enterprise Project    6 Comp Sci Electives
COMP9444 Neural Networks         6 Comp Sci Electives
COMP9517 Computer Vision         6 Comp Sci Electives
COMP9519 Multimedia Systems      6 Comp Sci Electives
COMP9596 Research Project       12 Comp Sci Electives
COMP9844 Ext Neural Networks     6 Comp Sci Electives
Free.... Free Electives (many   24 Free elect
GenEd... General Education (m   12 Gen Ed

