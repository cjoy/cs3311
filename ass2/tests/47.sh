# Student 3363921 studied 3978, COMPA1 starting in 10s1
php advisor 3363921 11s1
