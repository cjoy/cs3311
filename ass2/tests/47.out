Timothy Cappie-Wood (3363921)
Suggestions for study in 11s2

167 168Course   Title                 UOC Requirement
-------- -----                ---- -----------
